Python/Product/Debugger/Debugger/DebugEngine/DebuggerConstants.cs

```diff
@@ -65,6 +65,11 @@ class DebuggerConstants {
             get { return _guidLanguageDjangoTemplate; }
         }
 
+        static private Guid _guidLocalPortSupplier = new Guid("708C1ECA-FF48-11D2-904F-00C04FA302A1");
+        static public Guid guidLocalPortSupplier {
```

Comment:
@zooba, are all the new C# 6 features available for use? If so, this can be simplified to ` ... { get } = new Guid(...);`.